multicluster/charts/linkerd-multicluster/templates/gateway.yaml

```diff
@@ -32,6 +32,20 @@ spec:
       containers:
         - name: pause
           image: gcr.io/google_containers/pause
+          resources:
+            {{- toYaml .Values.gateway.resources | nindent 12 }}
```

Comment:
Can you wrap with `with` like you did below, so we don't end up with an empty section here with `.Values.gateway.resources` is not declared?